nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestGenerateTableFetch.java

```diff
@@ -207,6 +207,12 @@ public void testAddedRows() throws ClassNotFoundException, SQLException, Initial
         assertEquals("SELECT * FROM TEST_QUERY_DB_TABLE ORDER BY name OFFSET 4 ROWS FETCH NEXT 2 ROWS ONLY", new String(flowFile.toByteArray()));
         flowFile = runner.getFlowFilesForRelationship(REL_SUCCESS).get(3);
         assertEquals("SELECT * FROM TEST_QUERY_DB_TABLE ORDER BY name OFFSET 6 ROWS FETCH NEXT 2 ROWS ONLY", new String(flowFile.toByteArray()));
+        assertEquals("TEST_QUERY_DB_TABLE", flowFile.getAttribute("generatetablefetch.tableName"));
+        assertEquals(null, flowFile.getAttribute("generatetablefetch.columnNames"));
```

Comment:
All added asserts for columns are checking for null, can you update a test (or add one) that sets the Columns To Return property? Otherwise, looks good to me, will build and run on a real NiFi in the meantime